sites/public/src/pages/create-account.tsx

```diff
@@ -72,182 +73,339 @@ export default () => {
 
   return (
     <FormsLayout>
-      <FormCard>
-        <div className="form-card__lead text-center border-b mx-0">
-          <Icon size="2xl" symbol="profile" />
-          <h1 className="form-card__title">{t("account.createAccount")}</h1>
-          {requestError && (
-            <AlertBox className="" onClose={() => setRequestError(undefined)} type="alert">
-              {requestError}
-            </AlertBox>
-          )}
-          <SiteAlert type="notice" dismissable />
-        </div>
+      {process.env.showMandatedAccounts ? (
+        <Card>
+          <div className="form-card__lead pb-8 border-b mx-12 px-0">
+            <Icon size="2xl" symbol="profile" />
+            <Heading size="2xl" className="font-semibold mt-3">
+              {t("account.createAccount.title")}
+            </Heading>
+            {requestError && (
+              <AlertBox className="" onClose={() => setRequestError(undefined)} type="alert">
+                {requestError}
+              </AlertBox>
+            )}
+            <SiteAlert type="notice" dismissable />
+          </div>
+          <Form id="create-account" onSubmit={handleSubmit(onSubmit)}>
+            <div className="form-card__group border-b space-y-3 mx-12 p-0 py-8">
```

Comment:
Fully uptaking Card will require removing divs like this and using the Card sections. Reminder that there are examples in the application :)